lib/article_json/export/apple_news/elements/text.rb

```diff
@@ -14,6 +18,23 @@ class Text < Base
           def export
             super.to_s
           end
+
+          # @param [Nokogiri::XML::NodeSet] text
+          def create_text_nodes(text)
+            Nokogiri::HTML.fragment(sanitize_text(text).gsub(/\n/, '<br>')).children
```

Comment:
This looks a bit weird to me, why are there line breaks in the text? The Google Doc HTML export should already convert all those to `<br/>` tags, unless I'm missing something :thinking:

Reply:
Never mind, I just saw that the overwritten method does this too...